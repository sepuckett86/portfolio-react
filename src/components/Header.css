.Header {
  line-height: 40px;
  background-color: whitesmoke;
  display: flex;
  justify-content: space-between;
  align-items: center;

  ul {
    list-style: none;
    padding: 0;
    margin: 0;
    display: flex;
    flex-wrap: wrap;
    margin-right: 1.5em;
  }
  
  li {
    a {
      font-family: 'Open Sans', sans-serif;
      display: block;
      text-decoration: none; 
      color: #952024;
      padding-left: 1.5em;
      border-radius: 10px; 
    }
    a:hover {
      color: black;
    }
  }

  .name {
    margin-left: 1em;
    font-size: 1.3em;
    font-family: 'Open Sans', sans-serif;
    color: rgb(10, 146, 123);
  }
}