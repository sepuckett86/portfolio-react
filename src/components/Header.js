import React from 'react';
import styles from './Header.css';

// Only show when scrolled down. Also, position absolute??
function Header() {
  return (
    <header className={styles.Header}>
      <p className={styles.name}>Hello and welcome!</p>
      <ul>
        <li><a href="#portfolio">Portfolio</a></li>
        <li><a href="#contact">Contact</a></li>
      </ul>
    </header>
  );
}

export default Header;
