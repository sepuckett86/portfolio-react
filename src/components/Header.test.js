import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header component', () => {
  it('renders', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
