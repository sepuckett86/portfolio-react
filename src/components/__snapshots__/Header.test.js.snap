// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders 1`] = `
<header
  className="Header"
>
  <a
    className="name"
    href="/"
  >
    Hello and welcome!
  </a>
  <ul>
    <li>
      <a
        href="/#portfolio"
      >
        Portfolio
      </a>
    </li>
    <li>
      <a
        href="/#contact"
      >
        Contact
      </a>
    </li>
  </ul>
</header>
`;
